src/hooks/useDeployment.ts

```diff
@@ -0,0 +1,98 @@
+import { useSession } from "next-auth/react";
+import { useState, useCallback, useMemo } from "react";
+
+import { MaciService } from "~/services/maci";
+import { useEthersSigner } from "./useEthersSigner";
+import { eas } from "~/config";
+import * as vks from "../../public/vk_10-2-1-2_test.0.json";
+
+interface IDeployArgs {
+  amount: number;
+}
+
+export function useDeployment() {
```

Comment:
should we add a button on the page to go to the coordinator only page?

Reply:
sure, and the complete UI/UX flow would be another PR.

Reply:
resolve this conversation since would do this in another PR.